test/integration/auth.spec.ts

```diff
@@ -47,6 +50,7 @@ const mockUserData = {
   photoURL: 'http://www.example.com/' + newUserUid + '/photo.png',
   disabled: false,
 };
+let deleteQueue = Promise.resolve();
```

Comment:
const?

Reply:
I overwrite this at the bottom every time I queue a single delete:
```
  // Suppress errors in delete queue to not spill over to next item in queue.
  deleteQueue = deletePromise.catch((error) => {
    // Do nothing.
  });
  return deletePromise;
```